### ISSUE TITLE
Remote Content Browsing: Create 'My Downloads' Page

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#9861]
## Overview 

This issue is a front-end issue to add an "My Downloads" page that will display all content that has been downloaded to the device's Kolibri. 

### Figma 

Please see [complete figma spec ](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=194%3A2922&t=yAtcmSpF4FxfLCXa-0) for full specs.

<img width="462" alt="Screenshot 2023-01-31 at 9 10 32 AM" src="https://user-images.githubusercontent.com/17235236/215784144-a73e4122-98e2-4eac-967f-5dec1c07b25c.png">
<img width="508" alt="Screenshot 2023-01-31 at 9 10 48 AM" src="https://user-images.githubusercontent.com/17235236/215784158-45478573-4c7a-4f47-b514-f7b40ce90532.png">
<img width="526" alt="Screenshot 2023-01-31 at 9 11 46 AM" src="https://user-images.githubusercontent.com/17235236/215784189-01a393ae-9544-445a-9dd8-c59e2ad855e2.png">
<img width="575" alt="Screenshot 2023-01-31 at 9 15 19 AM" src="https://user-images.githubusercontent.com/17235236/215784397-95ee245f-844b-4a99-89cc-aca014d82f29.png">
<img width="771" alt="Screenshot 2023-01-31 at 9 17 30 AM" src="https://user-images.githubusercontent.com/17235236/215785383-f8fc0367-4dff-491d-a0d4-c821c4251297.png">
<img width="688" alt="Screenshot 2023-01-31 at 9 17 41 AM" src="https://user-images.githubusercontent.com/17235236/215785398-a5db9cc5-ad1e-42f7-984d-43e4de572c93.png">

## Acceptance Criteria
- [ ]  The "My downloads" page has been added to Kolibri, and is visible in the main navigation menu
- [ ]  A user can filter and sort the results 
- [ ] The downloads page is paginated in increments of 25 results
- [ ] A user can select one, multiple, or all results on the page to remove them from their downloads
- [ ] On "remove", the user is sees a confirmation modal
- [ ] Selection of results is persisted across pages
- [ ] The "Total size of downloads" can just be a sum of the file size data (i.e. we do not have to build around edge cases such as duplicate resources, rounding, etc.) 
- [ ] The "Free disk space" should be limited by the device settings. See [Device Settings Milestone](https://github.com/learningequality/kolibri/milestone/75) for context. 
- [ ] When a user wants to download a resource, use the (to be created) API endpoint that allows you to query if the resource is needed by anyone/anything other than this user. This is to account for a "remove download" scenario for 2+ user accounts on the same device, for example, where they have downloaded the same resource, and one person wants to remove it. The user should still be able to "delete" the resource from their downloads list, but the actual resource should remain on the device. The available space can update based on what the user sees.

### Currently blocked 
The following scenarios need design feedback before they can be implemented. 
1. Do we need the total size of "My Library"? (Probably not - skip implementing for now)
2. What should we display to a user when they downloaded a resource and garbage collection cleaned it up.
3. Modification to the UI and modal for the "a remove download scenario for 2+ user accounts" as detailed above

### Out of scope
Updates to the content renderer display are out of scope and covered in https://github.com/learningequality/kolibri/issues/9856

Please note that the updates the native app navigation are temporarily blocked by the App Bar updates, and can be done in follow up as needed.

## References 
- [Figma spec](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=194%3A2922&t=zDk6ybjN91p5hH6m-0)
- [Gherkin scenarios](https://www.notion.so/learningequality/Gherkin-scenarios-02096ac9925844f8902c818cab607d4b#0a205e75b2ea49a9be07acf15bf13b7a)

**Comment by rtibbles** [comment]
Implemented in #10137 - any follow up will be done in #10379
